### DISCUSSION TITLE
Change graphics settings

### DISCUSSION THREAD

**Discussion by oburato-dev** [korlibs/korge#2096] [Q&A] [answered]
Is there any simple way to change graphical settings such as maximum/minimum fps or antialiasing settings, full screen.. in runtime?

Whenever I try to change a setting it doesn't reflect in the game

**Comment by soywiz** [comment, marked as answer]
Fullscreen should be supported already. FPS is not AFAIK. Antialiasing I'm not sure but likely not. I have created an issue for that in any case.

https://github.com/korlibs/korge/issues/2103
